src/QualificationRunner.Core/Services/QualificationEngine.cs

```diff
@@ -128,7 +127,7 @@ private ExitCodes startBatchProcess(List<string> args, string cliPath, Cancellat
          using (var process = _startableProcessFactory.CreateStartableProcess(cliPath, args.ToArray()))
          {
             process.StartInfo.WindowStyle = ProcessWindowStyle.Hidden;
-            process.Start();
+            process.Start(ProcessPriorityClass.Idle);

```

Comment:
This reduces the process priority. It should not matter how many cores each process tries to use, the priority should always be lower than other apps the user is using.

For example, starting qualifications with population simulations should still play nice even though they may use all-but-one core each.